src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-wizard/rgw-multisite-wizard.component.html

```diff
@@ -0,0 +1,252 @@
+                    <div *ngFor="let realminfo of realms">
+                      <div class="form-group row">
+                        <label class="cd-col-form-label"
+                               for="realmName"
+                               i18n>Realm Name</label>
+                        <div class="cd-col-form-input">
+                          <input id="realmName"
+                                 name="realmName"
+                                 type="text"
+                                 [value]="realminfo.realm"
+                                 readonly>
+                          <cd-help-text>
+                            <span i18n>Name of the realm that will be involved in replication.</span>
+                          </cd-help-text>
+                        </div>
+                      </div>
+                      <div class="form-group row">
+                        <label class="cd-col-form-label"
+                               for="token"
+                               i18n>Token</label>
+                        <div class="cd-col-form-input">
+                          <input id="realmToken"
+                                 name="realmToken"
+                                 type="text"
+                                 [value]="realminfo.token"
+                                 class="me-2 mb-4"
+                                 readonly>
+                          <cd-copy-2-clipboard-button [source]="realminfo.token"
+                                                      [byId]="false">
+                          </cd-copy-2-clipboard-button>
+                          <cd-help-text>
+                            <span i18n>This field displays the token needed to import the multisite configuration into a secondary cluster. Copy this token securely and use it on the secondary cluster to replicate the current multisite setup. Ensure that the token is handled securely to prevent unauthorized access.</span>
+                          </cd-help-text>
+                        </div>
+                      </div>
+                      <hr *ngIf="realms.length > 1">
+                    </div>
+                  </ng-template>
+                </div>
+              </ng-container>
+            </form>
+          </div>
+        </div>
+        <div class="card-footer">
+          <button class="btn btn-accent m-2 float-end"
+                  (click)="onNextStep()"
+                  aria-label="Next"
+                  i18n>{{ showSubmitButtonLabel() }}
+            <span *ngIf="loading">
+              <i [ngClass]="[icons.spinner, icons.spin]"></i>
+            </span>
+          </button>
+          <cd-back-button class="m-2 float-end"
+                          aria-label="Close"
+                          (backAction)="onPreviousStep()"
+                          [name]="showCancelButtonLabel()"
+                          [disabled]="loading">
+          </cd-back-button>
```

Comment:
rather than disabling the cancel when there is something going on, which will be frustrating for the user that he is stuck staring at the spinner, let's provide a close button to close the wizard. But few things;
1. after he closes it, we should show a spinner (or a progress action) in the setup replication action button in the table.
2. and when the user clicks on the aciton button again, we directly show the progress and the logs or whatever to the him that way he can continue to do whatever he want and then come back and check the progress..

Reply:
Will take care of this in the progress page PR